source/deprecations/v1.x.html.md

```diff
@@ -1153,6 +1153,33 @@ Ember 1.11 made it possible to intuitively represent dynamic content in attribut
 
 This makes it possible to express a number of concepts directly in the template that previously were awkward to represent and required computer properties, and could even require `itemController`.
 
+##### Dasherized boolean values
+
+Ember 1.11 new syntax no longer supports dasherizing for boolean values. For example:
+
+```component
+// app/components/user-profile.js
+export default Ember.Component.extend({
+  isActiveUser: true
+});
+```
+
+```handlebars
+{{! app/templates/components/user-profile.hbs }}
+<div {{bind-attr class="isActiveUser"}}>
+</div>
+```
+
+Should be replaced with:
+
+```handlebars
+{{! app/templates/components/user-profile.hbs }}
+<div class="{{if isActiveUser 'is-active-user'}}">
+</div>
+```
+
+##### Legacy `bind-attr`
+
 Ember 1.13 deprecated `bind-attr` in favor of the new syntax.
```

Comment:
I wasn't really sure how to break it up. I didn't want to loose any of the content, but it seemed weird regardless of how I arranged it. Do you have any suggestions?